test/integration/v1/index.test.ts

```diff
@@ -0,0 +1,214 @@
+      apiEndpoint: process.env.FIREBOLT_API_ENDPOINT as string
+    });
+
+    await expect(async () => {
+      const connection = await firebolt.connect({
+        ...connectionParams,
+        engineName: "unknown_engine"
+      });
+      await connection.execute("SELECT 1");
+    }).rejects.toThrow();
+  });
+
+  it("fails on wrong engine url", async () => {
+    const firebolt = Firebolt({
+      apiEndpoint: process.env.FIREBOLT_API_ENDPOINT as string
+    });
+
+    const connection = await firebolt.connect({
+      auth: {
+        username: process.env.FIREBOLT_USERNAME as string,
+        password: process.env.FIREBOLT_PASSWORD as string
+      },
+      database: process.env.FIREBOLT_DATABASE as string,
+      engineEndpoint: "bad engine url"
+    });
+
+    await expect(async () => {
+      const statement = await connection.execute("SELECT 1");
+      await statement.fetchResult();
+    }).rejects.toThrow();
+  });
+  it("destroyed unfinished statements should throw", async () => {
+    const firebolt = Firebolt({
+      apiEndpoint: process.env.FIREBOLT_API_ENDPOINT as string
+    });
+    const connection = await firebolt.connect(connectionParams);
+    const st1 = connection.execute("SELECT 1");
+    const st2 = connection.execute("SELECT 2");
+    await connection.destroy();
+    expect(st1).rejects.toThrow("The user aborted a request.");
+    expect(st2).rejects.toThrow("The user aborted a request.");
+  });
+
+  it("stream", async () => {
+    const firebolt = Firebolt({
+      apiEndpoint: process.env.FIREBOLT_API_ENDPOINT as string
+    });
+
+    const connection = await firebolt.connect(connectionParams);
+
+    const statement = await connection.execute("SELECT * from numbers(100)", {
+      settings: {
+        output_format: OutputFormat.JSON_COMPACT,
+        use_standard_sql: 0
+      }
+    });
+
+    const {
+      data,
+      meta: metaPromise,
+      statistics: statisticsPromise
+    } = await statement.streamResult();
+
+    const rows: unknown[] = [];
+
+    data.on("metadata", metadata => {
+      console.log(metadata);
+    });
+
+    data.on("error", error => {
+      console.log("could be parse error", error);
+    });
+
+    const meta = await metaPromise;
+    console.log(meta);
+
+    for await (const row of data) {
+      rows.push(row);
+    }
+
+    const statistics = await statisticsPromise;
+    console.log(statistics);
+
+    expect(rows.length).toEqual(100);
+  });
+  // it.skip("format limited", async () => {
+  //   const firebolt = Firebolt({
+  //     apiEndpoint: process.env.FIREBOLT_API_ENDPOINT as string
+  //   });
+
+  //   const connection = await firebolt.connect(connectionParams);
+
+  //   const statement = await connection.execute("SELECT 1", {
+  //     settings: { output_format: OutputFormat.JSON_COMPACT_LIMITED }
+  //   });
+
+  //   const { data } = await statement.fetchResult();
+  //   const row = data[0];
+  //   expect(row).toMatchObject({ "1": 1 });
+  // });
```

Comment:
Any reason for this to be commented out?

Reply:
Removed it since it's skipped. We don't have `OutputFormat.JSON_COMPACT_LIMITED` as well as explicit support for output formats anymore